src/Symfony/Component/Validator/Test/ConstraintValidatorTestCase.php

```diff
@@ -573,9 +573,12 @@ public function expectNoValidate()
         $this->expectNoValidate = true;
     }
 
-    public function expectValidation(string $call, string $propertyPath, $value, $group, callable $constraints, ConstraintViolationInterface $violation = null)
+    public function expectValidation(string $call, ?string $propertyPath, $value, $group, callable $constraints, ConstraintViolationInterface $violation = null)
```

Comment:
better not change the signature and compare with `''` below, that'd remove any BC-breaking potential to this patch

Reply:
after a chat on Slack, this is legit